presto-docs/src/main/sphinx/functions/system.rst

```diff
@@ -0,0 +1,9 @@
+==================
+System Information
+==================
+
+Functions providing information about the system environment.
+
+.. function:: version() -> varchar
+
+    Returns the node version.
```

Comment:
Let’s call the this the server version, as “node” implies it is the version of a specific node (and which node in the cluster would it be?)